src/Raven.Server/Logging/RavenLogManagerServerExtensions.cs

```diff
@@ -504,7 +504,7 @@ public static void ConfigureAuditLog(this RavenLogManager logManager, RavenServe
         {
             Name = nameof(FileTarget),
             CreateDirs = true,
-            FileName = configuration.Security.AuditLogPath.Combine("${shortdate}.audit.log").FullPath,
+            FileName = configuration.Security.AuditLogPath.Combine("server.audit.log").FullPath,
```

Comment:
@hiddenshadow21 Could you check if we need a change in the other project around audit logs archivization routines please?

Reply:
Sorry, just noticed that the name of log file will be always the same. So we need to adjust to that. Otherwise we will overwrite old log file.